src/App.js

```diff
@@ -1,26 +1,47 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import ListView from './components/ListView';
+import AddView from './components/AddView';
+import Home from './components/Home';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  NavLink,
+} from 'react-router-dom';
 
-function App() {
+const App = () => {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Router>
+        <div>
+          <ul>
+            <li>
+              <NavLink to="/list-view" activeStyle={{ fontWeight: 'bold' }}>
+                ListView
+              </NavLink>
+            </li>
+            <li>
+              <NavLink to="/add-view" activeStyle={{ fontWeight: 'bold' }}>
```

Comment:
If you want to take this a step further, you could create an `active` class in the css and use `activeClassName` to set this equal to that class when the link is active. Then they will share the same css.

```js
              <NavLink to="/add-view" activeClassName="active">
```